doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst

```diff
@@ -193,11 +184,11 @@ The following are the prefixes used to indicate the cores:
    Peripheral Processor (PPR), ``ppr``, 0x2e
     , ``mod``, 0x24
```

Comment:
There is no core name for `mod` - cell in a table is empty.

Reply:
Internal docs did not define this. @hubertmis  Should we remove that line? @nordic-krch What exactly is "mod" in the logs?